src/features/meetings/part_duration/usePartDuration.tsx

```
@@ -150,7 +150,8 @@ const usePartDuration = ({ length, type, week }: PartDurationType) => {
       const source = sources.find((record) => record.weekOf === week);
 
       const time = sourcesPartTiming(source, type, dataView, lang);
-      setValue(time);
+
+      setValue(time || 5);
```

@FussuChalice: setting the default value could be misleading. If the custom part is really 5min then this will not be saved in the database since user action is required for save. For now, we can leave the empty duration to be default, and we will see if we really need to assign that in the future. Thanks.